lib/index.js

```diff
@@ -10,3 +12,6 @@ module.exports.tarball.stream = (spec, handler, opts) =>
   get(spec, opts).tarballStream(handler)
 module.exports.tarball.file = (spec, dest, opts) =>
   get(spec, opts).tarballFile(dest)
+module.exports.helpers = {
```

Comment:
Hm.  Creating a "helpers" object is a bit weird, kind of invites a grab-bag of stuff being exported without much rhyme or reason.  Maybe this should be a static method on the `pacote.Dir` class, and we could expose those classes, since it's really only used by the `Dir` class?